firebase-perf/src/main/java/com/google/firebase/perf/util/Rate.java

```diff
@@ -14,14 +14,16 @@
 
 package com.google.firebase.perf.util;
 
+import static java.util.concurrent.TimeUnit.SECONDS;
+
 import java.util.concurrent.TimeUnit;
 
 /** A Rate object representing the number of tokens per total specified time unit. */
 public class Rate {
 
-  private static final long NANO_IN_SECOND = 1000 * 1000 * 1000;
-  private static final long MICRO_IN_SECOND = 1000 * 1000;
-  private static final long MILLI_IN_SECOND = 1000;
+  private static final long NANO_IN_SECOND = SECONDS.toNanos(1);
```

Comment:
If instead of keeping these as constants we just use them inline, we could have more straightforward conversions. E.g. NANOSECONDS.toSeconds(...), MILLISECONDS.toSeconds(...), etc.

Reply:
I made then inline, but I don't think I can use NANOSECONDS.toSeconds(...) etc. because that converts nanoseconds to second, but we want to convert 1 second to nanosecond, for example. I kept the inline variables to SECONDS.toNanos(1), etc.